packages/itwinui-css/src/color-picker/color-picker.scss

```diff
@@ -175,8 +181,7 @@ $iui-hover-box-shadow: 0 0 0 var(--iui-size-3xs) var(--iui-color-border);
   forced-color-adjust: none;
   background-position: 0, 0, calc(var(--iui-size-xs) * 0.5) calc(var(--iui-size-xs) * 0.5);
   background-size: auto, var(--iui-size-xs), var(--iui-size-xs);
-  background-image:
-    linear-gradient(to right, transparent 0%, var(--iui-color-picker-selected-color) 100%),
+  background-image: linear-gradient(to right, transparent 0%, var(--iui-color-picker-selected-color) 100%),
```

Comment:
Ohh, in that case, it would be alright to leave these changes in. Because they might happen again in future PR when working on this file. So it's alright to commit these changes in this PR.

But if you want to avoid these changes (might be better since you didn't make any changes to this file), you can commit *just this file* with the `--no-verify` flag.